Document Email details stat cards and Pending card scope for Segment vs. triggered Emails (#782)
mautic/user-documentation: docs/channels/emails.rst

The Email overview allows at-a-glance information regarding the success or failure of a particular Email. You can quickly see relevant information in regards to opens, bounces, successful click-throughs and other important statistics.

> At the top of the Email details page, a row of stat cards summarizes how the Email is performing. Each card links to a filtered Contact list, so you can see exactly which Contacts it represents:

Reviewer: ```suggestion
.. vale off

At the top of the Email details page, a row of stat cards summarizes how the Email is performing. Each card links to a filtered Contact list, so you can see exactly which Contacts it represents:

.. vale on
```

- **Sent** - the number of Contacts that have received this Email, including both successful and failed sends, for the entire time period.
- **Read** - the number of Contacts that have read this Email, shown alongside the read percentage.
- **Queued** - the number of Contacts that have this Email queued to send based on their frequency rules.

> - **Pending** - the number of Contacts that haven't yet received this Email. This card appears only for Segment Emails. Triggered Emails - those used in a Campaign rather than sent to a Segment - don't show the **Pending** card. It always reported zero for them, which falsely suggested that Mautic was sending no Emails.

Author: [PR #16304](https://github.com/mautic/mautic/pull/16304) changes `details.html.twig` so the `mautic.email.stat.pending` card is only merged into `emailStats` when `email.isSegmentEmail()` is true, hiding it for triggered Emails where `getPendingCount()` always returned zero. This is the source for documenting that the **Pending** card appears only for Segment Emails. The stat definitions (Sent/Read/Queued/Pending) come from the `mautic.email.stat.*` tooltip strings in the EmailBundle `messages.ini`.

Source: https://github.com/mautic/mautic/pull/16304

> - **Pending** - the number of Contacts that haven't yet received this Email. This card appears only for Segment Emails. Triggered Emails - those used in a Campaign rather than sent to a Segment - don't show the **Pending** card. It always reported zero for them, which falsely suggested that Mautic was sending no Emails.

Reviewer: Follow our RST rule for unordered list.

```suggestion
* **Sent** - The number of Contacts that have received this Email, including both successful and failed sends, for the entire time period.
* **Read** - The number of Contacts that have read this Email, shown alongside the read percentage.
* **Queued** - The number of Contacts that have this Email queued to send based on their frequency rules.
* **Pending** - The number of Contacts that haven't yet received this Email. This card appears only for Segment Emails. Triggered Emails - those used in a Campaign rather than sent to a Segment - don't show the **Pending** card. It always reported zero for them, which falsely suggested that Mautic was sending no Emails.
```

.. vale off

At the top of the Email details page, a row of stat cards summarizes how the Email is performing. Each card links to a filtered Contact list, so you can see exactly which Contacts it represents:

.. vale on

* **Sent** - The number of Contacts that have received this Email, including both successful and failed sends, for the entire time period.
* **Read** - The number of Contacts that have read this Email, shown alongside the read percentage.
* **Queued** - The number of Contacts that have this Email queued to send based on their frequency rules.

> * **Pending** - The number of Contacts that haven't yet received this Email. This card appears only for Segment Emails. Triggered Emails - those used in a Campaign rather than sent to a Segment - don't show the **Pending** card given their send is scheduled differently or immediately sent.

Reviewer: @promptless-for-oss use active instead of passive voice. See 'is scheduled'.

> * **Pending** - The number of Contacts that haven't yet received this Email. This card appears only for Segment Emails. Triggered Emails - those used in a Campaign rather than sent to a Segment - don't show the **Pending** card. It always reported zero for them, which falsely suggested that Mautic was sending no Emails.

Reviewer 2: ```suggestion
* **Pending** - The number of Contacts that haven't yet received this Email. This card appears only for Segment Emails. Triggered Emails - those used in a Campaign rather than sent to a Segment - don't show the **Pending** card given their send is scheduled differently or immediately sent.
```

.. vale off

At the top of the Email details page, a row of stat cards summarizes how the Email is performing. Each card links to a filtered Contact list, so you can see exactly which Contacts it represents:

> 

Reviewer: Please add this image:

<img width="860" height="329" alt="Image" src="https://github.com/user-attachments/assets/02e31fac-a984-42dd-927d-4d26dfa7a6af" />

- Name it: email_stat_cards.png
- Add descripctive alt text. Don't start with "screenshot of" or anything alike.

.. vale off

> At the top of the Email details page, a row of stat cards summarizes how the Email is performing. Each card links to a filtered Contact list, so you can see exactly which Contacts it represents:

Reviewer: ```suggestion
At the top of the Email details page, a row of stat cards summarizes how the Email is performing. Each card links to a filtered Contact list, so you can see exactly which Contacts it represents.
```

> .. vale on

Reviewer: Move the image to before the unordered list, and remove the align center.

.. vale off

At the top of the Email details page, a row of stat cards summarizes how the Email is performing. Each card links to a filtered Contact list, so you can see exactly which Contacts it represents:

.. vale on

* **Sent** - The number of Contacts that have received this Email, including both successful and failed sends, for the entire time period.
* **Read** - The number of Contacts that have read this Email, shown alongside the read percentage.
* **Queued** - The number of Contacts that have this Email queued to send based on their frequency rules.
* **Pending** - The number of Contacts that haven't yet received this Email. This card appears only for Segment Emails. Triggered Emails - those used in a Campaign rather than sent to a Segment - don't show the **Pending** card because Mautic schedules them differently or sends them immediately.

.. vale off

.. image:: images/emails/email_stat_cards.png

>    :width: 600

Reviewer: Remove the width.
